pages/dkp/kommander/2.2/DKP-insight/index.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: layout.pug
+beta: false
+navigationTitle: DKP Insight
+title: DKP Insight
+excerpt: Use DKP insight to monitor and tune your enviornment
+menuWeight: 55
+beta: true
+enterprise: false
+---
+
+<p class="message--note"><strong>NOTE:</strong> Insights capability is offered with this release as a Technical preview only. Future enhancements include add-on licensing, a seamless user experience, and additional insight items.</p>
+
+DKP Insights is D2iQ’s latest analytics tool, designed to assist Kubernetes Administrators with daily tasks, such as making sure clusters are secure, debugging, monitoring, and anticipating future issues with your cluster. Insights provides you predictive analytics capability that provides greater support productivity, speed, and reduced cost. An Insights module, integrated into the DKP, collects configuration, logs, events, and metrics from DKP deployments and generates predictive insights on potential issues of varying critical levels so that you can quickly identify and resolve issues that would save a lot of time and money by not escalating to D2IQ's support.
```

Comment:
```suggestion
DKP Insights is D2iQ’s latest analytics tool, designed to assist Kubernetes Administrators with daily tasks, such as making sure clusters are secure, debugging, monitoring, and anticipating future issues with your cluster. Insights provides you predictive analytics capability that provides greater support productivity, speed, and reduced cost. An Insights module, integrated into the DKP, collects configuration, logs, events, and metrics from DKP deployments and generates predictive insights on potential issues of varying critical levels so that you can quickly identify and resolve issues that would save a lot of time and money by not escalating to D2iQ's support.
```

Reply:
Using comments above to work with.